pennylane/templates/transforms.py

```diff
@@ -0,0 +1,21 @@
+from functools import wraps

```

Comment:
@Thenerdstation can you add the copyright boilerplate at the top here, and also a module docstring?

Reply:
Done.